src/queueserver.ts

```diff
@@ -117,19 +117,19 @@ export interface IPlanSubmitState {
     readonly planLoading: boolean;
 }
 
-export const submitPlan = async(planId: number): Promise<IPlanObject> => {
+export const submitPlan = async(planId: number, param: number): Promise<IPlanObject> => {
     var planObj = {};
     if (planId === 0) {
         planObj = {
             name: "count",
             args: [["det1", "det2"]],
-            kwargs: {"num": 10, "delay": 1}
+            kwargs: {"num": param, "delay": 1}
```

Comment:
Should it be converted to int? I tried to run the plan with RE, and it failed:
```py
Run aborted
Traceback (most recent call last):
  File "/Users/mrakitin/anaconda3/envs/qs/lib/python3.8/site-packages/bluesky/run_engine.py", line 1365, in _run
    msg = self._plan_stack[-1].send(resp)
  File "/Users/mrakitin/anaconda3/envs/qs/lib/python3.8/site-packages/bluesky/plans.py", line 57, in count
    num_intervals = num - 1
TypeError: unsupported operand type(s) for -: 'str' and 'int'
```

Reply:
`param` should be of type `int` in this plan

Reply:
The plans are currently not annotated, so there is no type checking when plans are added to the queue.

Reply:
It is very strange, in my local testing I can see the quotes making it a string, I added the conversion to numeric type explicitly and the quotes go away. I am not sure why they are needed, but the simple two line fix results in a numeric type being used in the JSON submitted.

Reply:
Did the manager logs show the plan is executed after this change?

Reply:
It worked fine:
```
Transient Scan ID: 3     Time: 2020-10-20 16:05:17
Persistent Unique Scan ID: 'd886fca2-5eee-45a0-a494-1eda48fd63e8'
New stream: 'primary'
+-----------+------------+------------+------------+
|   seq_num |       time |       det1 |       det2 |
+-----------+------------+------------+------------+
|         1 | 16:05:17.8 |      5.000 |      1.765 |
|         2 | 16:05:18.8 |      5.000 |      1.765 |
|         3 | 16:05:19.8 |      5.000 |      1.765 |
|         4 | 16:05:20.8 |      5.000 |      1.765 |
|         5 | 16:05:21.8 |      5.000 |      1.765 |
|         6 | 16:05:22.8 |      5.000 |      1.765 |
|         7 | 16:05:23.8 |      5.000 |      1.765 |
|         8 | 16:05:24.8 |      5.000 |      1.765 |
|         9 | 16:05:25.8 |      5.000 |      1.765 |
|        10 | 16:05:26.8 |      5.000 |      1.765 |
|        11 | 16:05:27.8 |      5.000 |      1.765 |
|        12 | 16:05:28.8 |      5.000 |      1.765 |
|        13 | 16:05:29.8 |      5.000 |      1.765 |
|        14 | 16:05:30.8 |      5.000 |      1.765 |
|        15 | 16:05:31.8 |      5.000 |      1.765 |
|        16 | 16:05:32.8 |      5.000 |      1.765 |
|        17 | 16:05:33.8 |      5.000 |      1.765 |
|        18 | 16:05:34.8 |      5.000 |      1.765 |
|        19 | 16:05:35.8 |      5.000 |      1.765 |
|        20 | 16:05:36.8 |      5.000 |      1.765 |
+-----------+------------+------------+------------+
generator count ['d886fca2'] (scan num: 3)
```